src/index.html

```diff
@@ -1,13 +1,61 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <meta name="viewport" content="width=device-width,
+            user-scalable=no,
+            initial-scale=1.0,
+            maximum-scale=1.0,
+            minimum-scale=1.0">
   <title>Product cards</title>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="./style.css">
 </head>
+
 <body>
-  <h1>Product cards</h1>
+  <div class="product" data-qa="card">
+    <div class="product__image">
+      <img
+        src="images/imac.jpeg"
+        alt="Apple monitor"
+        class="product__picture"
+      >
+    </div>
+    <h3 class="product__title">
+      Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+    </h3>
+    <div class="product__code">
+      Код товара: 195434
+    </div>
+    <div class="product__rating">
+      <div class="product__stars stars--4">
+        <div class="product__star"></div>
```

Comment:
это элемент блока, лучше 
```suggestion
      <div class="stars stars--4">
        <div class="stars__star"></div>
```